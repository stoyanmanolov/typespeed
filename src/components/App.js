import React from "react";
import Home from "./Home";
import Race from "./Race";
import Ending from "./Ending";
import { HashRouter, Switch, Route } from "react-router-dom";

const App = () => {
  return (
    <HashRouter basename="/">
      <Switch>
        <Route exact path="/" component={Home} />
        <Route path="/race" component={Race} />
        <Route path="/end" component={Ending} />
      </Switch>
    </HashRouter>
  );
};

export default App;
