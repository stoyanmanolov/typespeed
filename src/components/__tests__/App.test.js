import React from "react";
import { shallow } from "enzyme";
import App from "../App";

describe("The App component", () => {
  let wrapper = shallow(<App />);

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
