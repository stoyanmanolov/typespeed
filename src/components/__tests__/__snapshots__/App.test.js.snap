// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App component renders correctly 1`] = `ShallowWrapper {}`;
